packages/cli/test/mocks/deployment.ts

```diff
@@ -7,17 +7,23 @@ import { Build, User } from '../../src/types';
 let deployments = new Map<string, Deployment>();
 let deploymentBuilds = new Map<Deployment, Build[]>();
 
+type State = Deployment['readyState'];
+
 export function useDeployment({
   creator,
+  state = 'READY',
 }: {
-  creator: Pick<User, 'id' | 'email' | 'name'>;
+  creator: Pick<User, 'id' | 'email' | 'name' | 'username'>;
+  state?: State;
 }) {
   const createdAt = Date.now();
   const url = new URL(chance().url());
+
   const deployment: Deployment = {
     id: `dpl_${chance().guid()}`,
     url: url.hostname,
-    name: '',
+    inspectorUrl: `https://vercel.com/${chance()}`,
+    name: chance.name,
```

Comment:
**suggestion (soft-blocking):** use a static arbitrary value instead

This always evaluates to `Chance`. I think you would need `chance().name()` to get a random name.

As above, I think we should avoid randomness in tests.

In this or a follow-up PR: Can this be a static arbitrary value instead?